guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-server.adoc

```diff
@@ -0,0 +1,56 @@
+[id="Renewing_a_Custom_SSL_Certificate_on_{project-context}_{context}"]
+= Renewing a {customssltitle} certificate on {ProjectServer}
+
+[NOTE]
+====
+If {ProjectServer} is already configured with CA-signed SSL certificates and has been renewed by the same Certificate Authority, you do not have to replace the consumer package `katello-ca-consumer-latest.noarch.rpm` on clients registered to {ProjectServer}.
+
+If the validity of CA-signed SSL certificates installed on {Project} from Certificate Authority A expires, and you installed an SSL certificate from Certificate Authority B, then you must re-deploy the `katello-ca-consumer-latest.noarch.rpm` package on every client registered to {Project}.
+This is because you require a new CA certificate for a client to authenticate or communicate with {Project}.
+====
+
+Use this procedure to update your renewed {customssl} certificate signed by a Certificate Authority (CA) for {ProjectServer}.
+The `katello-certs-check` command validates the input certificate files and returns the commands necessary to renew a {customssl} certificate for {ProjectServer}.
+
+ifdef::satellite[]
+.Prerequisite
+Refer to the {InstallingServerDocURL}Configuring_Server_with_a_Custom_SSL_Certificate_satellite[Configuring {ProjectServer} with a Custom SSL Certificate] guide before creating a new CSR because the Server certificate should have X.509 v3 `Key Usage` and `Extended Key Usage` extensions with required values.
+endif::[]
+.Procedure
+You must create a new Certificate Signing Request (CSR) and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {ProjectServer} certificate and CA bundle.
+
+Before deploying a renewed custom certificate on your {ProjectServer}, complete the following steps:
+
+* Validate the {customssl} input files.
+Note that for the `katello-certs-check` command to work correctly, Common Name (CN) in the certificate must match the FQDN of {ProjectServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c /root/{project-context}_cert/New_{project-context}_cert.pem \
+-k /root/{project-context}_cert/{project-context}_cert_key.pem \
+-b /root/{project-context}_cert/ca_cert_bundle.pem
```

Comment:
```suggestion
-b /root/_{project-context}_cert/ca_cert_bundle.pem_ \
-c /root/_{project-context}_cert/new_{project-context}_cert.pem_ \
-k /root/_{project-context}_cert/{project-context}_cert_key.pem_
```

I've sorted it in alphabetical order; used lowercase "new", and added italic highlighting to the paths that users have to replace/adjust.

Reply:
Even applied at other places :+1: